api.go

```diff
@@ -346,6 +353,58 @@ func (h serviceBrokerHandler) deprovision(w http.ResponseWriter, req *http.Reque
 	}
 }
 
+func (h serviceBrokerHandler) getBinding(w http.ResponseWriter, req *http.Request) {
+	vars := mux.Vars(req)
+	instanceID := vars["instance_id"]
+	bindingID := vars["binding_id"]
+
+	logger := h.logger.Session(getBindLogKey, lager.Data{
+		instanceIDLogKey: instanceID,
+		bindingIDLogKey:  bindingID,
+	})
+
+	versionCompatibility, err := checkBrokerAPIVersionHdr(req)
+	if err != nil {
+		h.respond(w, http.StatusPreconditionFailed, ErrorResponse{
+			Description: err.Error(),
+		})
+		logger.Error(apiVersionInvalidKey, err)
+		return
+	}
+	if versionCompatibility.Minor < 14 {
```

Comment:
Can we check Major too please? E.g. a 3.0 would fail here.

Reply:
The major version is checked in ``func checkBrokerAPIVersion()``. If the major version is anything other than ``2``, then an error is returned. See line 739 at https://github.com/pivotal-cf/brokerapi/pull/55/files#diff-651a84f8ad6a38c1f7ccdf63f2692410R739 . (This is similar to the code that was replaced by the PR.)

Given that the major version is always checked on entry to the various APIs, there's no point in rechecking it here. There are also unit-tests to verify that this is the case.